test/e2e-setup/bicep/no-cni.bicep

```diff
@@ -0,0 +1,49 @@
+@description('If set to true, the cluster will not be deleted automatically after few days.')
+param persistTagValue bool = false
+
+@description('Name of the hypershift cluster')
+param clusterName string
+
+module customerInfra 'modules/customer-infra.bicep' = {
+  name: 'customerInfra'
+  params: {
+    persistTagValue: persistTagValue
+  }
+}
+
+module managedIdentities 'modules/managed-identities.bicep' = {
+  name: 'managedIdentities'
+  params: {
+    clusterName: clusterName
+    vnetName: customerInfra.outputs.vnetName
+    subnetName: customerInfra.outputs.vnetSubnetName
+    nsgName: customerInfra.outputs.nsgName
+  }
+}
+
+module AroHcpCluster 'modules/cluster.bicep' = {
+  name: 'cluster'
+  params: {
+    clusterName: clusterName
+    vnetName: customerInfra.outputs.vnetName
+    subnetName: customerInfra.outputs.vnetSubnetName
+    nsgName: customerInfra.outputs.nsgName
+    userAssignedIdentitiesValue: managedIdentities.outputs.userAssignedIdentitiesValue
+    identityValue: managedIdentities.outputs.identityValue
+    networkConfig: {
+      networkType: 'Other'
```

Comment:
definitely looking forward to seeing this test.  Could the verification for this test including running two deployments that try to communication using pod-to-pod, pod-to-service, and pod-to-kube-apiserver?  We can probably publish some images from OCP to enable such a test.